### PR TITLE
Added Facebook Login sample [SDK-1478]

### PR DESCRIPTION
**Pull request by Widcket**
### Changes

This PR adds the Facebook Login sample and reorganizes the folders to match the native social QuickStarts changes. The new folder layout:
- **00-login-siwa** -> SIWA sample
- **00-login-facebook** -> Facebook Login sample

### References

This should be merged right after https://github.com/auth0/docs/pull/8848.

### CODE REVIEW

File: 00-login-facebook/Facebook/Info.plist [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	<key>CFBundlePackageType</key>
+	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>fb2668587536695558</string>
```
Comment (lbalmaceda) [inline 00-login-facebook/Facebook/Info.plist]: this shouldn't be here

File: 00-login-facebook/Facebook/Info.plist [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+			</array>
+		</dict>
+	</array>
+	<key>CFBundleVersion</key>
+	<string>1</string>
+	<key>FacebookAdvertiserIDCollectionEnabled</key>
+	<false/>
+	<key>FacebookAppID</key>
+	<string>2668587536695558</string>
```
Comment (lbalmaceda) [inline 00-login-facebook/Facebook/Info.plist]: this shouldn't be here

Comment (Widcket) [reply 00-login-facebook/Facebook/Info.plist]: Fixed in https://github.com/auth0-samples/auth0-ios-swift-siwa-samples/pull/8/commits/30a14ee0b1650b0e11b5146805fd23f5ba52c292.

File: 00-login-facebook/Facebook/Info.plist (hunk 1 above)
Comment (Widcket) [reply 00-login-facebook/Facebook/Info.plist]: Good catch! Fixed in https://github.com/auth0-samples/auth0-ios-swift-siwa-samples/pull/8/commits/30a14ee0b1650b0e11b5146805fd23f5ba52c292.

**Pull request merged**